tests/globalhelper/runhelper.go

```diff
@@ -47,5 +48,44 @@ func LaunchTests(testSuites []string, skipRegEx string) error {
 	cmd.Args = append(cmd.Args, testArgs...)
 	cmd.Dir = Configuration.General.TnfRepoPath
 
+	if os.Getenv("DEBUG_TNF") == "true" {
+		os.Setenv("LOG_LEVEL", "trace")
```

Comment:
Done.

Reply:
I believe that's also a bit confusing, Nikita, as usually TNF refers to the test-network-function suite that this QE repo is testing. I know we should change our tnf LOG_LEVEL variable into TNF_LOG_LEVEL, but yours should be something like QE_LOG_LEVEL or TNF_QE_LOG_LEVEL.
Basically, my recommendation is that in your QE repo, TNF/tnf prefix should always refer to the test-network-function test suite names to avoid confusion, and you should use something like QE/qe prefixes to your variables/function when distinguish them from the context is not so straightforward.

Reply:
Agree. @shaior Please re-name this env var to VERIFICATION_LOG_LEVEL. @greyerof What do you think?